core/jazz_ui/src/app/pages/environment-deployment/env-deployments-section.component.ts

```diff
@@ -307,8 +309,14 @@ export class EnvDeploymentsSectionComponent implements OnInit {
             this.buildNo[i] = this.deployments[i].provider_build_id;
             this.buildurl[i] = this.deployments[i].provider_build_url;
             this.deployment_id[i] = this.deployments[i].deployment_id;
-            this.backupLogs = this.deployments
+            this.backupLogs = this.deployments;
             this.sort = new Sort(this.deployments);
+            if(this.deployments[i].status == "started"){
+              countStarted = countStarted + 1;
+            }
+           }
+           if(countStarted == 0){
```

Comment:
if(!countStarted)

Reply:
it will not work ... becoz i'm initalizing it as 0

Reply:
That should not be an issue.